.github/workflows/test.yml

```diff
@@ -67,33 +68,36 @@ jobs:
             ]
         cabal: ['3.6']
         include:
-          # only test supported ghc major versions
-          - os: ubuntu-latest
-            ghc: '9.0.1'
-            test: true
-          - os: ubuntu-latest
-            ghc: '8.10.7'
-            test: true
-          - os: ubuntu-latest
-            ghc: '8.8.4'
-            test: true
-          - os: ubuntu-latest
-            ghc: '8.6.5'
-            test: true
-          - os: windows-latest
-            ghc: '9.0.1'
-            test: true
-          - os: windows-latest
-            ghc: '8.10.7'
-            test: true
-          - os: windows-latest
-            ghc: '8.6.5'
-            test: true
-          # only build rest of supported ghc versions for windows
-          - os: windows-latest
-            ghc: '8.10.6'
-          - os: windows-latest
-            ghc: '8.8.4'
+           # only test supported ghc major versions
+           - os: ubuntu-latest
+             ghc: '9.2.1'
+             test: true
+           - os: ubuntu-latest
+             ghc: '9.0.1'
+             test: true
+           - os: ubuntu-latest
+             ghc: '8.10.7'
+             test: true
+           - os: ubuntu-latest
+             ghc: '8.8.4'
+             test: true
+           - os: ubuntu-latest
+             ghc: '8.6.5'
+             test: true
+          # Blocked on unix-compat
```

Comment:
I added a version of Unix compat in the cabal-ghc921.project to enable the windows build, and it worked for me locally

Reply:
I can't look into Windows fixes, please send it

Reply:
I did it before the force push 🙂 with 2a2738507fd62cc21fd5fd38d2d80199d542df60  but will redo

Reply:
In fact all changes are lost, will try to recover them cherry-picking my local branch